apps/state/lib/state/service.ex

```diff
@@ -75,11 +85,17 @@ defmodule State.Service do
           _ -> {service.start_date, service.end_date, service.days}
         end
 
+      attributes = Enum.find(calendar_attributes, fn s -> s.service_id == service_id end)
+
       %Service{
         id: service_id,
         start_date: start_date,
         end_date: end_date,
         valid_days: valid_days,
+        description: attributes && attributes.description,
+        schedule_name: attributes && attributes.schedule_name,
+        schedule_type: attributes && attributes.schedule_type,
+        schedule_typicality: (attributes && attributes.schedule_typicality) || 0,
```

Comment:
You could simplify this code by assigning `attributes` as an empty `%Parse.CalendarAttributes{}` in the case where the ID isn't found?